doc/OnlineDocs/modeling_extensions/gdp/modeling.rst

```diff
@@ -203,16 +203,16 @@ Integration with Disjunctions
 
 .. note::
 
-    This section is subject to change between this date and August 2020.
-    The current plan is to provide a deprecation path changing ``indicator_var`` from binary ``Var`` to ``BooleanVar``.
+    Historically, the ``indicator_var`` on ``Disjunct`` objects was
+    implemented as a binary ``Var``.  Beginning in Pyomo 6.0, that has
+    been changed to the more mathematically correct ``BooleanVar``, with
+    the associated binary variable available as
+    ``binary_indicator_var``.
```

Comment:
Can someone create a `Disjunct` and then link the `BooleanVar` to a different binary? This is the difference between `disjunct.binary_indicator_var` and `disjunct.indicator_var.get_associated_binary()` right? But with the current implementation `disjunct.binary_indicator_var` would still always be right, even in this case, correct?